### PR TITLE
tests: Improved TestGamma and TestTheta u-tests

### PR DESCRIPTION
**Pull request by sjalander**

### CODE REVIEW

File: tests/libjlm/backend/hls/rvsdg2rhls/TestGamma.cpp [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #include <jive/view.hpp>
 
 #include <jlm/backend/hls/rvsdg2rhls/rvsdg2rhls.hpp>
```
Comment (phate) [inline tests/libjlm/backend/hls/rvsdg2rhls/TestGamma.cpp:13]: Could this header be removed?

File: tests/libjlm/backend/hls/rvsdg2rhls/TestTheta.cpp [hunk 2]
```diff
@@ -9,6 +9,7 @@
 #include <jive/rvsdg/theta.hpp>
 
 #include <jlm/backend/hls/rvsdg2rhls/rvsdg2rhls.hpp>
```
Comment (phate) [inline tests/libjlm/backend/hls/rvsdg2rhls/TestTheta.cpp:11]: The same here. Could this header be removed?

**Comment by phate** [comment]
Could we rename the tests to TestGammConversion.cpp and TestThetaConversion.cpp?

File: tests/libjlm/backend/hls/rvsdg2rhls/TestTheta.cpp (hunk 2 above)
Comment (sjalander) [reply tests/libjlm/backend/hls/rvsdg2rhls/TestTheta.cpp]: Removed

File: tests/libjlm/backend/hls/rvsdg2rhls/TestGamma.cpp (hunk 1 above)
Comment (sjalander) [reply tests/libjlm/backend/hls/rvsdg2rhls/TestGamma.cpp]: Removed

**Pull request merged**